e2e/cypress/integration/api/iam/projects_api_ingestion.spec.ts

```diff
@@ -209,12 +270,32 @@ describeIfIAMV2p1('Ingestion project tagging', () => {
       });
     });
 
+    // Ingest an action with attributes that match all the projects
+    cy.fixture('action/environment_create.json').then((action) => {
+      action.organization_name = 'Team Pizza';
+      action.remote_hostname = 'example.pizza';
+      action.id = actionId;
+      action.entity_name = entityName;
+      action.recorded_at = Cypress.moment().utc().subtract(1, 'day').format();
+
+      cy.request({
+        headers: { 'api-token': Cypress.env('ADMIN_TOKEN') },
+        method: 'POST',
+        url: '/data-collector/v0',
+        body: action
+      });
+    });
+
     // wait for the report to be ingested
     cy.waitForNodemanagerNode(complianceNodeId, 30);
-    cy.waitForComplianceNode(complianceNodeId, start, end, 30);
+    cy.waitForComplianceNode(complianceNodeId, nodeStart, nodeEnd, 30);
+
     // wait for the client run report to be ingested
     cy.waitForNodemanagerNode(clientRunsNodeId, 30);
     cy.waitForClientRunsNode(clientRunsNodeId, 30);
+
+    // wait for the action to be ingested
+    cy.waitForAction(entityName, 30);
```

Comment:
If you change this to `cy.waitForAction(entityName, eventStart, eventEnd, 30);` and add the two below lines above the test pass for me.

```
  const eventEnd = Cypress.moment().utc().endOf('day').valueOf().toString();
  const eventStart = Cypress.moment().utc().subtract(3, 'day').valueOf().toString();
```